Formula/csvtk.rb

```diff
@@ -0,0 +1,17 @@
+class Csvtk < Formula
+  desc "A cross-platform, efficient and practical CSV/TSV toolkit in Golang"
+  homepage "http://bioinf.shenwei.me/csvtk"
+  url "https://github.com/shenwei356/csvtk/archive/refs/tags/v0.23.0.tar.gz"
+  sha256 "b8ccf2b86c815693af1c7d743ca711a68fcdde94d9a9279ac8557fdffa3f2fc8"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-s -w", *std_go_args
+  end
+
+  test do
+    system "#{bin}/#{name}"
```

Comment:
We need a test that exercises some functionality of the program, that isn't as simple as running it, printing the help text or comparing the version. Please check [this section](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) of the Formula Cookbook for more details.

Perhaps, in this case, you would want to parse a small CSV file and check the output of the `csvtk` command with relevant subcommands/options.

Reply:
Sure! The repo itself provides some tests that I could use. Can I download files during the test phase? How can I do that in the formula?

Reply:
If you want to download a CSV file from the `testdata` folder of the repository, you need to use a `resource` block.

You can take a look at [`marcli`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/marcli.rb) for an example. Note that you'll have to pin the resource to a commit rather than `main`/`master`, as we want to avoid checksum mismatches if there's any change in the future.